src/testTest.test.ts

```diff
@@ -0,0 +1,3 @@
+it("works", () => {
+  expect(true).toBe(true);
+})
```

Comment:
basic plain/non-react test. there's [tons of "matchers"](https://jestjs.io/docs/using-matchers), such as `toBe`, `toEqual`, `toBeUndefined`, etc etc)